README.md

```diff
@@ -89,6 +88,20 @@ When running on a unix machine, provide the path to the 'bin' directory.
 > ./gradlew clean build-info-extractor-nuget:test
 ```
 
+#### Docker tests
+* Docker tests run only on Linux/mac.
+* In addition to general environment variables you must set the following docker environment variables:
```

Comment:
In addition to general environment variables you must set the following docker environment variables:
-->
In addition to the general environment variables required for running the tests, you must set the following environment variables, required for the docker tests: